react/features/base/devices/middleware.js

```diff
@@ -129,7 +146,7 @@ function _checkAndNotifyForNewDevice(store, newDevices, oldDevices) {
             description,
             titleKey,
             customActionNameKey: 'notify.newDeviceAction',
-            customActionHandler: _useDevice.bind(undefined, store, newDevice)
+            customActionHandler: _useDevice.bind(undefined, store, newDevices)
```

Comment:
I think maybe passing in all the new devices is causing problems with setting the preferred device. I think the array might be all devices.

Reply:
Oh, I was renaming some stuff and retesting ... maybe I broke it this should be devicesArray